app/controllers/api/subcollections/snapshots.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module Subcollections
+    module Snapshots
+      def snapshots_query_resource(object)
+        object.snapshots
+      end
+
+      def snapshots_create_resource(parent, _type, _id, data)
+        parent.snapshots.create!(data.merge(:create_time => Time.zone.now))
```

Comment:
@abellotti do you know if the `create_time` is intended to be set by the user as a required field?